docs/docs/evaluation/pipeline.md

```diff
@@ -0,0 +1,114 @@
+# Pipelines
+
+You can use pipelines to run a series of commands together.
+This is useful for automating evaluation tasks.
+
+You can also use pipelines to run multiple commands in sequence in a CI environment.
+For example, you might want to run a pipeline of commands every day.
+
+## Run Pipelines
+
+You can run pipelines with the [`runPipeline()`](../reference/evaluation/modules.md#runpipeline) function.
+
+`runPipeline()` takes the following arguments:
+
+- `configConstructor`: A [`ConfigConstructor`](../reference/evaluation/modules.md#configconstructor) function that returns a configuration object.
+- `pipeline`: A [`Pipeline`](../reference/evaluation/modules.md#pipeline) function that defines the pipeline based on actions in the configuration.
+
+Example `runPipeline()` function call with its `ConfigConstructor` and `Pipeline`:
+
+```ts
+// somePipeLine.ts
```

Comment:
```suggestion
```ts title="somePipeLine.ts"
```

Docusaurus code blocks have title support built in

Reply:
oh good to know. we use the same pattern employed here with the file name as a comment throughout a few places in docs so will keep same for consistency for now. but very open to change all going forward...good intern task. will make ticket